pyvista/plotting/plotter.py

```diff
@@ -1281,69 +1284,73 @@ def disable_anti_aliasing(self, all_renderers=True):
             self.renderer.disable_anti_aliasing()
 
     @wraps(Renderer.set_focus)
-    def set_focus(self, *args, render=True, **kwargs):  # numpydoc ignore=PR01,RT01
+    def set_focus(self, *args, render=True, **kwargs) -> None:  # numpydoc ignore=PR01,RT01
         """Wrap ``Renderer.set_focus``."""
         log.debug('set_focus: %s, %s', str(args), str(kwargs))
         self.renderer.set_focus(*args, **kwargs)
         if render:
             self.render()
 
     @wraps(Renderer.set_position)
-    def set_position(self, *args, render=True, **kwargs):  # numpydoc ignore=PR01,RT01
+    def set_position(self, *args, render=True, **kwargs) -> None:  # numpydoc ignore=PR01,RT01
         """Wrap ``Renderer.set_position``."""
         self.renderer.set_position(*args, **kwargs)
         if render:
             self.render()
 
     @wraps(Renderer.set_viewup)
-    def set_viewup(self, *args, render=True, **kwargs):  # numpydoc ignore=PR01,RT01
+    def set_viewup(self, *args, render=True, **kwargs) -> None:  # numpydoc ignore=PR01,RT01
         """Wrap ``Renderer.set_viewup``."""
         self.renderer.set_viewup(*args, **kwargs)
         if render:
             self.render()
 
     @wraps(Renderer.add_orientation_widget)
-    def add_orientation_widget(self, *args, **kwargs):  # numpydoc ignore=PR01,RT01
+    def add_orientation_widget(
+        self, *args, **kwargs
+    ) -> _vtk.vtkOrientationMarkerWidget:  # numpydoc ignore=PR01,RT01
         """Wrap ``Renderer.add_orientation_widget``."""
         return self.renderer.add_orientation_widget(*args, **kwargs)
 
     @wraps(Renderer.add_axes)
-    def add_axes(self, *args, **kwargs):  # numpydoc ignore=PR01,RT01
+    def add_axes(self, *args, **kwargs) -> AxesActor:  # numpydoc ignore=PR01,RT01
         """Wrap ``Renderer.add_axes``."""
         return self.renderer.add_axes(*args, **kwargs)
 
     @wraps(Renderer.add_box_axes)
-    def add_box_axes(self, *args, **kwargs):  # numpydoc ignore=PR01,RT01
+    def add_box_axes(self, *args, **kwargs) -> AxesActor:  # numpydoc ignore=PR01,RT01
```

Comment:
This can't be right, `AxesActor` is the regular axes with arrows, not the box.  I this return type may be coming from the docstring for this method, which also says `AxesActor` is returned (and which also isn't right). 

`vtk.vtkAnnotatedCubeActor` should probably be the return type here.

I'm surprised `mypy` is even allowing this...

Reply:
Fixed, thanks

Reply:
> I'm surprised mypy is even allowing this..

I think the wrapped renderer method also needs to be typed for mypy to check the internal code here. Otherwise it will be typed as `Any` and always pass.